typify-impl/Cargo.toml

```diff
@@ -17,7 +17,7 @@ schemars = "0.8.21"
 semver = "1.0.23"
 serde = "1.0.215"
 serde_json = "1.0.133"
-syn = { version = "2.0.89", features = ["full"] }
+syn = { version = "2.0.89", features = ["full", "visit-mut"] }
```

Comment:
Is this new feature only needed for tests?

Reply:
Yes, good catch, I moved that feature to the `dev-dependencies`.